Extensions/TweenBehavior/shifty.d.ts

```diff
@@ -0,0 +1,571 @@
+// Shifty.js 2.16.0 type definitions by arthuro555
```

Comment:
I had actually begun a PR to autogenerate typings from the jsdoc comments 👀 Since it requires changing those comments quite a lot though, and for some reason i cannot get jsdoc to work, I was afraid

Reply:
Ah, I understand. There might be a few pitfalls in there, but I'm sure we can get them resolved. I'd be happy to help out where I can, so just let me know what I can do. Even just getting a PR going with what you've started on will help the process along quite a bit! 🙂